l293x/l293x.go

```diff
@@ -0,0 +1,100 @@
+// Package l293x provides a driver to the L293/L293D H-bridge chip
+// typically used to control DC motors.
+//
+// Datasheet: https://www.ti.com/lit/ds/symlink/l293d.pdf
+//
+package l293x // import "tinygo.org/x/drivers/l293x"
+
+import (
+	"machine"
+)
+
+// Device is a motor without speed control.
+type Device struct {
+	d1, d2 machine.Pin
+	power  machine.Pin
+}
+
+// New returns a new Motor driver for GPIO-only operation.
+func New(d1, d2, p machine.Pin) Device {
```

Comment:
I'd change p => power so it's more clear what the function expect

Reply:
Did some renaming based on your suggestions, and also matching the datasheet more closely.